src/bin/nydusd/fs_cache.rs

```diff
@@ -416,16 +418,49 @@ impl FsCacheHandler {
             match self.create_data_blob_object(&config, msg.fd) {
                 Err(s) => format!("copen {},{}", hdr.msg_id, s),
                 Ok((blob, blob_size)) => {
-                    state
-                        .id_to_object_map
-                        .insert(hdr.object_id, (FsCacheObject::DataBlob(blob), msg.fd));
+                    state.id_to_object_map.insert(
+                        hdr.object_id,
+                        (FsCacheObject::DataBlob(blob.clone()), msg.fd),
+                    );
                     state.id_to_config_map.insert(hdr.object_id, config.clone());
+                    let _ = self.do_prefetch(&config, blob);
                     format!("copen {},{}", hdr.msg_id, blob_size)
                 }
             }
         }
     }
 
+    pub fn do_prefetch(&self, config: &BlobCacheConfigDataBlob, blob: Arc<dyn BlobCache>) {
+        let blob_info = config.blob_info().deref();
+        let factory_config = config.factory_config().deref();
+        if !factory_config.cache.prefetch_config.enable {
+            return;
+        }
+        let size = factory_config.cache.prefetch_config.merging_size as u64;
+        let blob_size=blob_info.compressed_size();
+        let count = blob_size / size + 1;
```

Comment:
When blob_size could be divided by size, count will be one larger than expected. So should it be
```
let count = (blob_size + size - 1) / size
```
?